responsibleai/text/components/rai_text_insights/spec.yaml

```diff
@@ -1,59 +1,75 @@
 $schema: http://azureml/sdk-2-0/CommandComponent.json
 name: rai_text_insights
 display_name: RAI Text Insights
-version: 0.0.7
+version: 0.0.8
 tags:
   Preview: ""
 type: command
 inputs:
   task_type:
     type: string # [text_classification]
     enum: ['text_classification', 'multilabel_text_classification']
+    description: The type of task to perform
   model_input: # mlflow model name:version
     type: mlflow_model
     optional: false
+    description: The model to be used for computing insights
   model_info:
     type: string # model name:version
     optional: false
+    description: The model to be used for computing insights
   test_dataset:
     type: mltable
+    description: The test dataset to be used for computing insights
   target_column_name:
     type: string
+    description: The target column name
   text_column_name:
     type: string
     optional: true
     default: ''
+    description: Optional text column name, required if metadata present
   maximum_rows_for_test_dataset:
     type: integer
     default: 5000
+    description: The maximum number of rows to use from the test dataset
   classes:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The class names for the target column
   categorical_metadata_features:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The list of categorical metadata feature names
   dropped_metadata_features:
     type: string # Optional[List[str]]
     default: '[]'
+    description: The list of dropped metadata feature names
   enable_explanation:
     type: boolean
     default: True
+    description: Whether to enable computing explanations
```

Comment:
Maybe reword to "Whether to enable computation of explanations"?

Reply:
done!